### PR TITLE
Fixes exceptions when there's a rest exception

### PR DESCRIPTION
**Pull request by nesquena**
A few fixes when there's an exception, the lack of error handling causes exceptions in certain cases.

### CODE REVIEW

File: lib/mini_fb.rb [hunk 1]
```diff
@@ -675,10 +674,14 @@ def self.fetch(url, options={})
 
             return res_hash
         rescue RestClient::Exception => ex
-            puts "ex.http_code=" + ex.http_code.to_s
-            puts 'ex.http_body=' + ex.http_body if @@logging
-            res_hash = JSON.parse(ex.http_body) # probably should ensure it has a good response
-            raise MiniFB::FaceBookError.new(ex.http_code, "#{res_hash["error"]["type"]}: #{res_hash["error"]["message"]}")
+          puts "ex.http_code=" + ex.http_code.to_s
```
Comment (blanchma) [inline lib/mini_fb.rb]: Add if @logging here too

File: lib/mini_fb.rb [hunk 2]
```diff
@@ -675,10 +674,14 @@ def self.fetch(url, options={})
...
+          puts 'ex.http_body=' + ex.http_body if @@logging
```
Comment (blanchma) [inline lib/mini_fb.rb]: Log both: code and body in the same puts and don't forget "if @logging"

**Comment by blanchma** [comment]
Rebase and +1

**Comment by treeder** [comment]
@blanchma can you fix this and give me another pull request so I can merge it?

**Pull request closed without merging**